api-v2/models/database.py

```diff
@@ -0,0 +1,20 @@
+from psycopg2 import connect 
+from psycopg2.extras import RealDictCursor #Access DB fields access columns only from keys
+
+class Database(object):
+    def __init__(self, app=None):
+        self.app = app
+        if app is not None:
+            self.init_app(app)
+    
+    def init_app(self, app):
+        self.conn = connect("host='localhost' dbname='ridemyway' user='postgres' password='Secrets'")
```

Comment:
add these variables to a `dot-env` file. It is not a good practice to push your connection details to Github.

Reply:
@esirK 
On it. Thanks